src/components/menu/menu-item/menu-item.component.js

```diff
@@ -170,7 +169,7 @@ const MenuItem = ({
 
   return (
     <StyledMenuItem
-      role="presentation"
+      data-component="menu-item"
```

Comment:
I think need this on the `StyledMenuItem` on line 142 as well, for a menu item with a submenu